install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -308,7 +308,7 @@ spec:
       sourceOwner: source.owner | ""
       destinationApp: destination.labels["app"] | ""
       destinationIp: destination.ip | ip("0.0.0.0")
-      destinationServiceHost: destination.service.host | ""
+      destinationServiceHost: destination.service.host | request.host | ""
```

Comment:
for TCP `request.host` is never set afaik.

Reply:
this should be removed too...

Reply:
Thanks! Done.